### ISSUE TITLE
tests

### ISSUE THREAD

**Issue by lookfirst** [swimlane-contrib/systemjs-route-bundler#6]
this project really needs some tests. =)

**Comment by amcdnl** [comment]
Agreed.  Want to handle some while we figure out some of the last holes?

**Comment by lookfirst** [comment]
part of my issue is that i have no idea what this code does. it is some serious magic for me. i can start to try to get the test framework setup?

**Comment by lookfirst** [comment]
ok... working on setting up tests now...

**Comment by marjan-georgiev** [comment]
I'm writing a blog post explaining what this does. Should have something by the end of the week. Will let you know.

**Comment by lookfirst** [comment]
ok... just added PR #8 

the treewalker is where most of the real magic is... added an empty spec for that...

**Comment by lookfirst** [comment]
There is also a path issue. `gulp test` in the seed project doesn't work. the issue is finding the bower_components folder as defined as the 'paths' part of `system.config.js`. The problem is that when we are running a bundle, we need it to be `../bower_components`... when we run tests, it needs to just be `./bower_components`. Not really sure how to fix that yet. =(

**Comment by marjan-georgiev** [comment]
We fixed that in our app today. I updated the seed project. Test framework and 'gulp test' should be working now.

**Comment by lookfirst** [comment]
Couple problems with your fix... 
1. `jspm install` reorders the system.config.js::map, so it shows up changed in git
2. `gulp test` now errors out in gulp-less (update your various npm packages)...

```
angular-systemjs-seed/node_modules/gulp-less/index.js:68
    }).done(undefined, cb);
       ^
TypeError: Object #<Promise> has no method 'done'
```

**Comment by lookfirst** [comment]
updating gulp-less to 3.0.0 fixes that issue

**Comment by lookfirst** [comment]
`chai` also doesn't belong in the `system.config.js`
